### ISSUE TITLE
Trouble formatting predictions for DSEC-Disparity website submission

### ISSUE THREAD

**Issue by SchuckLee** [uzh-rpg/DSEC#80]
Hello,I'm working on submitting predictions from a monocular depth estimation (MDE) network to the DSEC-Disparity benchmark. My model **outputs a log-transformed depth map,** and I currently use the following function to convert it back to a linear depth before submission:

```
def from_log_to_depth(input_log_image, Dmax=128, alpha=3.7):
    depth_image = Dmax * torch.exp(-alpha * (1 - input_log_image))
    return depth_image
```

Following [SCSNet](https://github.com/Chohoonhee/SCSNet), I use Dmax=128. After converting, I multiply the output by 256 and save it as a .png file using uint16 format, as required. Here's a snippet of my post-processing:

```
logits_pred = self.model(img, mode="whole", rescale=True)
disparity = from_log_to_depth(logits_pred[0])  # torch tensor
disparity_np = disparity.squeeze().cpu().numpy()
disparity_uint16 = (disparity_np * 256.0).astype(np.uint16)
cv2.imwrite(save_path, disparity_uint16)
```
However, although the visualization looks fine and validation on my own dataset gives good performance, the DSEC submission evaluation returns poor metrics:

1PE: 98.13
2PE: 96.27
MAE: 35.464
RMSE: 43.717

I would like to ask:
What is the expected depth or disparity range for submission? Should it be in [0, 80], or do you expect raw disparities?
Is there any normalization or clipping expected before multiplying by 256?
Am I correct in assuming that I should output disparity (not depth) maps?
Any guidance would be greatly appreciated! Thanks for your great work on the DSEC benchmark.
